mix.exs

```diff
@@ -1,24 +1,32 @@
 defmodule GGity.MixProject do
   use Mix.Project
 
+  @source_url "https://github.com/srowley/ggity"
+  @version "0.3.0"
+
   def project do
     [
       app: :ggity,
-      version: "0.3.0",
+      version: @version,
       aliases: aliases(),
       elixir: "~> 1.10",
       start_permanent: Mix.env() == :prod,
       package: package(),
       deps: deps(),
       dialyzer: [plt_add_apps: [:mix]],
       name: "GGity",
-      description: "GGity brings the familiar interface of R's ggplot2 library to SVG charting in Elixir.",
-      source_url: "https://github.com/srowley/ggity",
+      description: """
+        GGity brings the familiar interface of R's ggplot2 library to SVG
+        charting in Elixir.
+      """,
       homepage_url: "http://www.pocketbookvote.com",
       licenses: "MIT",
       docs: [
-        extras: ["README.md", "ROADMAP.md"],
-        main: "readme"
+        source_url: @source_url,
+        source_ref: "v#{@version}",
+        extras: ["README.md", "ROADMAP.md", "CHANGELOG.md"],
+        main: "readme",
+        assets: "assets"
```

Comment:
See comment above about putting images in the docs. 

Thanks for adding `:source_ref` - I was not familiar with that!

Reply:
> See comment above about putting images in the docs.
> 
> Thanks for adding `:source_ref` - I was not familiar with that!

:+1: 
This will have to match the `@version` and the GitHub release tag.

Reply:
Your blog post sold me on putting it in `:docs`! That was a great forensic adventure. 😀

Also, it looks like you deleted the `:licenses` key, was that intentional?

Reply:
> Your blog post sold me on putting it in `:docs`! That was a great forensic adventure. 
> 
> Also, it looks like you deleted the `:licenses` key, was that intentional?

The `:licenses` option should be put under the `:package` option.

See https://hex.pm/docs/publish. 
> Under the :package property are some additional configuration options:
> ......
> :licenses    A list of licenses the project is licensed under. This attribute is required. It is recommended to use SPDX License identifier.